datavec/pom.xml

```diff
@@ -262,9 +278,17 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
+                            <environmentVariables>
+                                <CUDA_VISIBLE_DEVICES>0</CUDA_VISIBLE_DEVICES>
+                            </environmentVariables>
                             <parallelMavenExecution>false</parallelMavenExecution>
                             <parallel>false</parallel>
                             <forkCount>0</forkCount>
+                            <threadCount>1</threadCount>
+                            <perCoreThreadCount>false</perCoreThreadCount>
+                            <forkCount>0</forkCount>
+                            <threadCount>1</threadCount>
+                            <perCoreThreadCount>false</perCoreThreadCount>
```

Comment:
I think you've copied too much here.

Reply:
In general, the goal with cuda is to prevent any form of multi threading.  Could you elaborate on what bits?